zerver/openapi/zulip.yaml

```diff
@@ -21025,6 +21025,17 @@ paths:
                   example: false
                 message_retention_days:
                   $ref: "#/components/schemas/MessageRetentionDays"
+                is_archived:
+                  description: |
+                    A boolean indicating whether the channel is [archived](/help/archive-a-channel)
+                    or unarchived.
+
+                    **Changes**: In Zulip 11.0 (feature level {ZF-7ec2f5}), added as a parameter
+                    to PATCH /stream/{stream_id}. Previously, we were not able to
```

Comment:
There is no need to have this explaination here. Just mentioning "New in Zulip 11.0 (feature leve ..." should be fine.